src/alchemtest/gmx/expanded_ensemble/case_3/descr.rst

```diff
@@ -0,0 +1,22 @@
+Gromacs: Host CB7 and Guest C3 in water
+=========================
+
+Notes
+-----
+Data Set Characteristics:
+    :Number of Legs: 2 (Coulomb, VDW)
+    :Number of Windows: 32 total, 20 for Coulomb, 12 for VDW
+    :Number of Simulations: 32
+    :Length of Simulation: 5ns
+    :System Size: 8286 atoms
+    :Temperature: 300 K
+    :Alchemical Pathway: vdw + coul --> vdw --> vacuum
+    :Missing Values: None
+    :Creator: \T. Jensen
+    :Donor: Travis Jensen (travis.jensen@colorado.edu)
+    :Date: November 2017
+    :License: `CC0 <https://creativecommons.org/publicdomain/zero/1.0/>`_ Public Domain Dedication 
+	      
+Host CB7 and Guest C3 in water, Guest C3 alchemically turned into Guest C3 in vacuum separated from water and Host CB7
```

Comment:
Do you have a paper that should be cited with the data? You can include it in the same way as in the Amber test data https://raw.githubusercontent.com/alchemistry/alchemtest/master/src/alchemtest/amber/simplesolvated/descr.rst

Reply:
The data generated has not been used in a publication but the system was part of the SAMPL4 challenge and similar simulations on this system were performed by a student of our group for a publication. Would you suggest citing one of these?

Reply:
In general I like context. You could state where the systems came from (citing the appropriate SAMPL4 host/guest overview paper... assuming that there is one) and saying that the data are similar to the ones in <cite paper from your group>.

Reply:
> However, I am not aware of a publication that used replica exchange on this system.

You can always say that these are unpublished data.

My primary concern is simply that right now is probably the only time ever that someone (i.e., you) is looking closely at the data here and the associated meta-data so now is the time to provide any useful information. In principle it is not super-important to know everything because you are the expert who is going to write the tests anyway and you know your system. However, if anyone else wants to use the test data to test their own implementations then this additional information can come in handy.

Use your own judgement. In principle I am happy with the PR.